### DISCUSSION TITLE
Downloading Broadcasts from X

### DISCUSSION THREAD

**Discussion by starboarded** [aclap-dev/video-downloadhelper#294] [Help and Q&A]
### Your question

Hello, 
I'm wondering if it's possible to download a broadcast from X. Or, if there's any other app that might be able to do that.
I specifically want to download this `https://twitter.com/i/broadcasts/1LyxBnOvzvOxN` but I don't want to have to record it for an hour and a half.
Thanks!

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: no.

**Comment by franxov** [comment]
i found a method that works. add the stream detector extension to your browser. then open the broadcast that you wish to download. the stream detector will provide URLs that will allow you to download them. copy one of the URLs and paste them in any downloader such as JDownloader, ytdlp etc.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

A lot has changed since this discussion was opened about 10 months ago.  Today, my environment is Windows 11 Pro 64-bit version 24H2 build 26100.2894, Firefox 134.0.1 64-bit, licensed VDH 9.1.0.41 beta, CoApp 2.0.20.

The URL mentioned in the original post is now downloaded without problem with VDH.  No need for any additional browser extensions & fiddling around with other downloaders.  VDH works on it now.
